docs/api/v3.rst

```diff
@@ -472,140 +522,230 @@ Build triggering
-            "next": "/api/v3/projects/pip/subprojects/?limit=10&offset=10",
+            "next": "/api/v3/projects/pip/redirects/?limit=10&offset=10",
             "previous": null,
-            "results": ["PROJECT"]
+            "results": ["REDIRECT"]
         }
 
-    :>json integer count: total number of projects.
-    :>json string next: URI for next set of projects.
-    :>json string previous: URI for previous set of projects.
-    :>json array results: array of ``project`` objects.
+Redirect create
++++++++++++++++
 
-    :requestheader Authorization: token to authenticate.
+.. http:post:: /api/v3/projects/pip/redirects/
 
+    Create a redirect for this project.
 
-Translations
-~~~~~~~~~~~~
+    **Example request**:
 
-Translations listing
-++++++++++++++++++++
+    .. sourcecode:: bash
 
+        $ curl \
+          -X POST \
+          -H "Authorization: Token <token>" https://readthedocs.org/api/v3/projects/pip/redirects/ \
+          -H "Content-Type: application/json" \
+          -d @body.json
 
-.. http:get:: /api/v3/projects/(str:project_slug)/translations/
+    The content of ``body.json`` is like,
 
-    Retrieve a list of all translations for a project.
+    .. sourcecode:: json
+
+        {
+            "from_url": "/docs/",
+            "to_url": "/documentation/",
+            "type": "page"
+        }
+
+    **Example response**:
+
+    `See Redirect details <#redirect-details>`_
+
+    :requestheader Authorization: token to authenticate.
+
+    :statuscode 201: Created
+    :statuscode 400: Some field is invalid
+
+
+Redirect update
++++++++++++++++
+
+.. http:put:: /api/v3/projects/(str:project_slug)/redirects/(int:redirect_id)/
+
+    Update a redirect for this project.
 
     **Example request**:
 
     .. sourcecode:: bash
 
-        $ curl -H "Authorization: Token <token>" https://readthedocs.org/api/v3/projects/pip/translations/
+        $ curl \
+          -X PUT \
+          -H "Authorization: Token <token>" https://readthedocs.org/api/v3/projects/pip/redirects/1/ \
+          -H "Content-Type: application/json" \
+          -d @body.json
 
-    **Example response**:
+    The content of ``body.json`` is like,
 
     .. sourcecode:: json
 
         {
-            "count": 25,
-            "next": "/api/v3/projects/pip/translations/?limit=10&offset=10",
-            "previous": null,
-            "results": ["PROJECT"]
+            "from_url": "/docs/",
+            "to_url": "/documentation.html",
+            "type": "page"
         }
 
-    :>json integer count: total number of projects.
-    :>json string next: URI for next set of projects.
-    :>json string previous: URI for previous set of projects.
-    :>json array results: array of ``project`` objects.
+    **Example response**:
+
+    `See Redirect details <#redirect-details>`_
 
     :requestheader Authorization: token to authenticate.
+
+    :statuscode 200: Success
+    :statuscode 400: Some field is invalid
+
+
+Redirect destroy
```

Comment:
I think `delete` is probably more common than `destroy` -- we should standardize on one.

Reply:
He, yeah. The verb is DELETE but DRF uses destroy so :man_shrugging: 

I think delete makes more sense here.